examples/usecases/multi-gpu-data-parallel-training.ipynb

```diff
@@ -0,0 +1,459 @@
+{
```

Comment:
<p>may be rephrase:  <code>We will use the same dataset and preprocessing steps with NVTabular</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/855/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>